### PR TITLE
Catch and display project validation exceptions correctly

### PR DESCRIPTION
**Pull request by pakrym**
Issue: https://github.com/aspnet/dnx-watch/issues/20
@victorhurdugaci

### CODE REVIEW

File: src/Microsoft.Dnx.Watcher.Core/Impl/ProjectProvider.cs [hunk 1]
```diff
@@ -12,19 +12,28 @@ public class ProjectProvider : IProjectProvider
...
 
-            errors = null;
-            project = new Project(runtimeProject);
+            try
+            {
+                project = new Project(runtimeProject);
+            }
+            catch (Exception ex)
+            {
+                errors = ex.Message;
```
Comment (victorhurdugaci) [inline src/Microsoft.Dnx.Watcher.Core/Impl/ProjectProvider.cs]: You should also include messages from the inner exception (if any). Also, you might want to handle `AggregateExceptions` as a special case

**Comment by pakrym** [comment]
:up: :date:

File: src/Microsoft.Dnx.Watcher.Core/Impl/ProjectProvider.cs [hunk 2]
```diff
@@ -43,11 +53,33 @@ private bool TryGetProject(string projectFile, out Runtime.Project project, out
...
             }
 
             project = null;
             return false;
         }
+
+        private string CollectMessages(Exception exception)
+        {
+            var builder = new StringBuilder();
+            builder.Append(exception.Message);
```
Comment (victorhurdugaci) [inline src/Microsoft.Dnx.Watcher.Core/Impl/ProjectProvider.cs]: Use AppendLine everywhere

**Comment by victorhurdugaci** [comment]
:shipit:

**Pull request merged**